specs/SensitivityLabel.md

```diff
@@ -0,0 +1,541 @@
+interface ICoreWebView2SensitivityLabelEventArgs : IUnknown {
+  /// Gets a read-only collection of all sensitivity labels detected in the
+  /// current web document. This collection contains instances of sensitivity
+  /// labels that have been reported by the web page.
+  [propget] HRESULT SensitivityLabels(
+      [out, retval] ICoreWebView2SensitivityLabelCollectionView** value);
+
+
+  /// Gets the current state of sensitivity label detection.
+  [propget] HRESULT SensitivityState(
+      [out, retval] COREWEBVIEW2_SENSITIVITY_LABEL_STATE* value);
+
+}
+
+/// Receives `SensitivityLabelChanged` events.
+[uuid(927a011d-bbf3-546b-ba28-1fc0ef4c1f4a), object, pointer_default(unique)]
+interface ICoreWebView2SensitivityLabelChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2* sender,
+      [in] ICoreWebView2SensitivityLabelEventArgs* args);
+}
+
+/// Extension of the ICoreWebView2 interface that provides sensitivity label
+/// change notification capabilities. This interface enables applications to
+/// monitor and respond to changes in the sensitivity classification of web
+/// content loaded in the WebView2 control. When sensitivity labels are
+/// detected, updated, or removed from web pages, the SensitivityLabelChanged
+/// event is raised.
+[uuid(ac4543d5-f466-5622-8b3b-24d3b195525c), object, pointer_default(unique)]
+interface ICoreWebView2_32 : IUnknown {
+  /// Adds an event handler for the `SensitivityLabelChanged` event.
+  /// Event raised when the sensitivity label classification of web page 
+  /// changes. web pages may report sensitivity labels via
+  /// [`Page Interaction Restriction Manager`](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PageInteractionRestrictionManager/explainer.md).
+  /// This event is triggered when the WebView2 control detects a change in 
+  /// the sensitivity labels associated with the currently loaded web page.
+  /// Changes can occur when navigating to a new page in the main frame,
+  /// when the existing page updates its sensitivity label information.
+  /// On navigation to a new page `SensitivityLabelChanged` event is raised
+  /// after the `NavigationCompleted` event. Applications can subscribe to
+  /// this event to receive notifications about sensitivity changes. The
+  /// event args provide the current label state and the complete collection
+  /// of detected sensitivity labels. the complete collection of detected
+  /// sensitivity labels.
+  HRESULT add_SensitivityLabelChanged(
+      [in] ICoreWebView2SensitivityLabelChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with 
+  /// `add_SensitivityLabelChanged`.
+  HRESULT remove_SensitivityLabelChanged(
+      [in] EventRegistrationToken token);
+
+}
+```
+### .NET/WinRT
+```c#
+enum CoreWebView2SensitivityLabelKind
```

Comment:
This kind of enum can be dangerous because new values are "emitted" from the API without apps opting in to new values.  (unlike enums used as API parameters, where the app must have recompiled with a new behavior change to start interacting with the new value.)

Be very explicit in the documentation that new values may be added here and show up on the eventargs.  We should at least discourage consumers from coding up a "default" case that assumes they knew all the possible label kinds when they wrote the code.